api/src/main/java/marquez/service/models/Job.java

```diff
@@ -42,6 +44,8 @@ public Job(
       @NonNull final JobId id,
       @NonNull final JobType type,
       @NonNull final JobName name,
+      @NonNull String simpleName,
```

Comment:
Do we have to update the UI to use the _simple name_ when displaying the job name?